src/types/resourceTypes.ts

```diff
@@ -63,7 +64,7 @@ export interface IndicatorValue {
 // Indicator values for a singe feature area - a dictionary of indicator ids to  values
 export type FeatureIndicatorValues = Dict<IndicatorValue>;
 
-export type MapFeature = Feature<Polygon | MultiPolygon>;
```

Comment:
so the feature type accepts generics like the type of geometry `Polygon | MultiPolygon` and the properties we expect them to have. Typescript requires you to specify all or no generics, which means it assumed the properties were of type `never` and that threw errors in my ide since we are accessing properties. So I just specified the most general type of property here, we could do our specific one with `ID_0`, `COUNTRY`, etc. but i feel like thatll be more effort than its worth to keep up if the scientists keep changing the names